pkg/node/node.go

```diff
@@ -223,3 +237,30 @@ func (n *Node) ClusterDNS() string {
 	}
 	return "10.100.0.10"
 }
+
+// EKSResourceAccount returns the AWS account id that provides node resources for EKS
+func (n *Node) EKSResourceAccount() string {
+	switch n.Region {
+	case "ap-east-1":
+		return eksResourceAccountAPEast1
+	case "me-south-1":
+		return eksResourceAccountMESouth1
+	default:
+		return eksResourceAccountStandard
+	}
+}
+
+// ContainerArchitecture returns a normalised architecture name as used in
+// container image names. If the architecture is unknown: defaults to `amd64`
+func (n *Node) ContainerArchitecture() string {
```

Comment:
I think this bit here was better before, because now when you'll add more architectures you'll have to keep appending `&&` conditions to this one if statement and it'll end-up a bit messy. It is also a bit less explicit on the intention, I think. Regardless, it's okay for now since it only supports these 2 architectures.

Reply:
I can't think that AWS are likely to add another arch anytime soon... this is KISS for now!